caktus/etlman
# Fixes #88: Add `sql_query` to `DataInterface` model

Adds an **sql_query** field to the `Data Interface` model.

Test:

1. Click the Add Pipeline button
2. Ensure that the field displays
3. Fill out the form and then click edit (ensure that the correct content displays in the field)
4. Verify content in Django Admin
5. Run tests locally

Closes #88
